### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by ioandr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,62 @@
-# invenio-statistics-js
-Invenio-Statistics client library for producing D3 graphs based on statistics.
+## Invenio-Statistics-Js
+
+[![Release](https://img.shields.io/github/tag/inveniosoftware/invenio-search-js.svg)](https://github.com/inveniosoftware/invenio-search-js/releases)
+[![LICENSE](https://img.shields.io/github/license/inveniosoftware/invenio-search-js.svg)](https://github.com/inveniosoftware/invenio-statistics-js/blob/master/LICENSE)
+
+Invenio-Statistics is a reusable client library for integrating charts into modern web apps. All charts are built with D3.js.
+
+## Table of contents
```
Comment (switowski) [inline README.md]: I think you can remove the `Table of contents` and the `Dependencies` sections (they just take space of the page).

File: examples/tutorials/1_started.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Tutorial: Getting started
+
+## General
+`Invenio-Statistics-Js` helps you visualize statistics expressed in `JSON` format.
+
+This module is designed to cooperate with the `Invenio-Stats` module, in order to visualize CDS statistics
```
Comment (switowski) [inline examples/tutorials/1_started.md]: You could add the URL of the `invenio-stats` module.

**Review by switowski — COMMENTED** [review]
How about we add screenshots to the tutorials, so it's immediately visible how the final result will look like?

Also, I see that you are nicely documenting each configuration option in the tutorials, but maybe it would be also useful to see ALL the possible configuration options that we might need in one place (unless the options from the tutorial are those `ALL options we will need` :) )? WDYT @drjova ?

**Comment by drjova** [comment]
Closed by 6625402ed5e88610a19aa8c94944683c3267171d

**Pull request closed without merging**